react/ui-components/src/atoms/stories/Accordion.stories.js

```
@@ -131,32 +98,27 @@ WithNumber.args = {
+NestedAccordion.args = {
   isOpenInitially: false,
   hideCardBorder: false,
   hideDivider: false,
   hideCardBg: false,
   hideBorderRadius: false,
+  title: "Main Accordion",
+  children: (
+    <Accordion
+      title={"Nested Accordion"}
+      children={
+        "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged."
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)